README.md

```diff
@@ -641,6 +642,14 @@ Other keys used to sign some previous releases:
 * **Timothy J Fontaine** &lt;tjfontaine@gmail.com&gt;
 `7937DFD2AB06298B2293C3187D33FF9D0246406D`
 
+## License
+
+Node.js is available under the
+[MIT license](https://opensource.org/licenses/MIT). Node.js also includes
```

Comment:
This should link to the `LICENSE` file instead.

Reply:
The license file is linked just below, are you suggesting to also remove the next sentence?

Reply:
ping @jasnell  I think this is the last thing we need to clarify before this can land.

Reply:
Yes, removing the last sentence and moving the link to the `LICENSE` file should work.

Reply:
@jasnell sorry I missed your comment before landing, do you want me to revert?